core/ts.core/src/ts/client/ITypeScriptServiceClient.java

```diff
@@ -91,8 +92,8 @@
 	 * @param insertString
 	 * @throws TypeScriptException
 	 */
-	void changeFile(String fileName, int line, int offset, int endLine, int endOffset, String insertString)
-			throws TypeScriptException;
+	void changeFile(String fileName, int line, int offset, int endLine, int endOffset,

```

Comment:
I had the correct preferences configured, but somehow the format on save did not include some lines. I'm double-checking that now.